### PR TITLE
DFR-927: fix nightly security scan issue

### PR DESCRIPTION
**Pull request by rishikrsharma**
Fix security scan script

### CODE REVIEW

File: security.sh [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 export LANG=C.UTF-8
-zap-cli --zap-url http://0.0.0.0 -p 1001 report -o /zap/api-report.html -f html
-zap-cli --zap-url http://0.0.0.0 -p 1001 alerts -l High --exit-code False
-cp /zap/api-report.html functional-output/
-cat zap.out
\ No newline at end of file
+export PYTHONDONTWRITEBYTECODE=1
+
+echo "Run ZAP scan and generate reports"
+zap-api-scan.py -t ${URL_FOR_SECURITY_SCAN}/v2/api-docs -f openapi -S -d -u ${SECURITY_RULES} -P 1001 -l FAIL --hook=zap_hooks.py -J report.json -r api-report.html
```
Comment (gokul-sol) [inline security.sh:8]: need to replace ${URL_FOR_SECURITY_SCAN} with ${TEST_URL} variable on line 8 and ${SECURITY_RULES} should be camel case like this ${SecurityRules}

File: security.sh [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+echo "Print alerts"
+zap-cli --zap-url http://0.0.0.0 -p 1001 alerts -l Informational --exit-code False
+
+echo "LC_ALL: ${LC_ALL}"
+echo "LANG: ${LANG}"
+echo "PYTHONDONTWRITEBYTECODE: ${PYTHONDONTWRITEBYTECODE}"
+echo "Print zap.out logs:"
+cat zap.out
+echo "Copy artifacts for archiving"
+cp zap.out functional-output/
```
Comment (gokul-sol) [inline security.sh:18]: Might need this line after line 18
cp report.json functional-output/

as report.json is passed as param in line 8.. probably need to copy the output of it

File: security.sh (hunk 1 above)
Comment (rishikrsharma) [reply security.sh]: Thank you @gokul-sol . Done.

File: security.sh (hunk 2 above)
Comment (rishikrsharma) [reply security.sh:18]: done

**Review by anatolysan — APPROVED** [review]

**Pull request merged**